app/templates/base.html

```diff
@@ -21,6 +21,8 @@
                 margin-left: 10%;;
```

Comment:
```suggestion
                margin-left: 10%;
```
I understand that this is unrelated, just might be worthwhile catching this to save time.